packages/react-instantsearch/src/components/RefinementList.js

```diff
@@ -34,8 +29,19 @@ class RefinementList extends Component {
     limitMin: PropTypes.number,
     limitMax: PropTypes.number,
     transformItems: PropTypes.func,
+    className: PropTypes.string,
   };
 
+  static defaultProps = {
+    className: '',
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = { query: '' };
```

Comment:
can't we use the `state = { query: '' };` pattern, since class fields are transpiled anyway